sdcm/cluster.py

```diff
@@ -4704,6 +4709,24 @@ def install_scylla_bench(node):
             GO111MODULE=on go get -v github.com/scylladb/scylla-bench@v0.1.2
         """))
 
+    @staticmethod
+    def install_cassandra_harry(node):
+        if node.distro.is_rhel_like:
+            node.remoter.sudo("yum install -y git make maven")
+        else:
+            node.remoter.sudo(shell_script_cmd("""\
+                apt-get update
+                apt-get install -y git make maven
+            """))
+        node.remoter.run(shell_script_cmd("""\
+            rm -rf cassandra-harry
+            git clone -b scylla-branch https://github.com/amoskong/cassandra-harry
```

Comment:
hook it to specific version/commit_id

Reply:
I think it's not needed, it's better to use latest and if needed to fix it.

Reply:
@roydahan , we have had same issue with all of the loaders, one day it is getting updated without us noticing it, test begin to fail, no one knows what is going on, after hours/days of investigation it turns out that it is just a tool got upgraded.